packages/gatsby-theme-docs/src/layouts/internals/footer.js

```diff
@@ -3,8 +3,7 @@ import { useStaticQuery, graphql } from 'gatsby';
 import styled from '@emotion/styled';
 import SpacingsInline from '@commercetools-uikit/spacings-inline';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
-import { designSystem, MediaQuery } from '@commercetools-docs/ui-kit';
-import { LogoHorizontalSvgIcon } from '../../icons';
+import { designSystem, MediaQuery, Icons } from '@commercetools-docs/ui-kit';
```

Comment:
Same for the Icons, they are not exported "flat" in the UI Kit namespace any more for clarity

Reply:
FYI: this is technically a breaking change, as the package exports changed.

Something to keep in mind for the next time.

Reply:
Also to keep in mind that grouping exports into one object might cause some issues with tree shaking. Maybe it's not the case here but better be careful with that.

Reply:
dammit, yes - completely obvious that this is all a breaking change (assuming we had external users). 

Concerning the grouping / tree shaking, I think we need to address that once more than the current one and later likely two implementations use it.  For now it's just about decoupling gatsby bound things from non-gatsby-things and keeping it monolithic by design.